src/commands/deployments/viewCommitInGitHub.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DeploymentTreeItem } from "vscode-azureappservice";
+import { ext } from "../../extensionVariables";
+
+export async function viewCommitInGitHub(node?: DeploymentTreeItem): Promise<void> {
+    if (!node) {
+        node = <DeploymentTreeItem>(await ext.tree.showTreeItemPicker('deployment/github'));
```

Comment:
Just to confirm - the casing here doesn't matter?

Reply:
The casing here does matter.  In the DeploymentTreeItem constructor, I made everything `toLowerCase()` to avoid confusion.  Maybe this should be done in the TreeItemPicker level?

Reply:
Yeah TreeItemPicker should probably ignore case if it's not already. Not a big deal, though